core/src/project.rs

```diff
@@ -84,6 +84,7 @@ typsy::call! {
     }
 }
 
+#[allow(clippy::suspicious_operation_groupings)]
```

Comment:
Is it possible to force clippy to lint this even though it's allowed?  99% of the time we're going to want to allow all these lints, but once in a while we might want to completely audit the code, to see if the lints still make sense, and the easiest way would be to tell clippy to ignore all allows.

Reply:
That's what this `allow` does. It tells `clippy` to ignore these lints for the given block. We can then grep for `allow(clippy::` to find them all. I've added this one in particular because I have tests for `is_overlapping` that cover all the cases, so I know it's correct.

Reply:
Yeah, I was hoping for a switch, something like `cargo clippy --deny-allowed`.  Laziness on my part! :smile: